Tests/Functional/Domain/Repository/FootnoteRepositoryTest.php

```diff
@@ -159,10 +160,17 @@ public function shouldThrowExceptionWithInvalidObject()
 
     /**
      * @test
+     * assert that no exception is thrown
      */
     public function shouldAddObject()
     {
-        $footnote = new Footnote();
-        $this->repository->add($footnote);
+        try {
+            $footnote = new Footnote();
+            $this->repository->add($footnote);
+        } catch (IllegalObjectTypeException $notExpected) {
```

Comment:
shouldn't this test fail if any exception is thrown?

Reply:
Yes.. But the Expection Type has to be the same.. So could be that the exception isn't as expected.

Reply:
correct - any exception should fail this test - changed :-)